src/bench/streams_findbyte.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+
+#include <fs.h>
+#include <streams.h>
+
+static void FindByte(benchmark::Bench& bench)
+{
+    // Setup
+    FILE* file = fsbridge::fopen("streams_tmp", "w+b");
+    const size_t file_size = 200;
+    uint8_t b = 0;
+    for (size_t i = 0; i < file_size; ++i) {
+        fwrite(&b, 1, 1, file);
+    }
+    b = 1;
+    fwrite(&b, 1, 1, file);
+    rewind(file);
+    CBufferedFile bf(file, file_size * 2, file_size, 0, 0);
```

Comment:
It's super small either way, but I don't think the buffer needs to be twice the file size?
```suggestion
    CBufferedFile bf(file, file_size + 1, file_size, 0, 0);
```